analyzer/src/test/java/com/novoda/sqlite/StringUtilTest.java

```diff
@@ -0,0 +1,21 @@
+package com.novoda.sqlite;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class StringUtilTest {
+    @Test
+    public void shouldCamelizeColumnNames(){
+        assertEquals("ColumnId", StringUtil.camelify("COLUMN_ID"));
+        assertEquals("Columnid", StringUtil.camelify("columnID"));
+        assertEquals("Columnid", StringUtil.camelify("COLUMNID"));
```

Comment:
@blundell Yes, these tests reflect the current text conversions we support. It's targeted at all uppercase words separated by underscore, and does not play well in other cases. The tests highlight that. 
Might be good to have separated test cases for them though, as we'll have to do more work on that.